src/pages/Category.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+
+import ReviewItem from '../components/ReviewItem';
+// import './category.css';
+
+export default function Category() {
+
+    const [categoryReviews, setCategoryReviews] = useState([]);
+    const category = useParams();
+
+    useEffect(() => {
+        axios.get(`https://shadia-nc-games.herokuapp.com/api/reviews/?category='${category.slug}'`)
+            .then(({ data }) => setCategoryReviews(data.reviews))
+            .catch((err) => {
+                console.log('error', err)
+            })
+    }, []);
```

Comment:
Would be better if you put your api requests in a single file, then they're all together. Like the lecture on Monday. 

Also, i'd look at using the axios params object to have queries like this. You should be able to use the same api request that you used for the get all reviews one